internal/store/static.go

```diff
@@ -140,6 +140,11 @@ func (store *StaticStore) getPlugins(serverVersion string, includeEnterprisePlug
 		}
 
 		if serverVersion != "" && storePlugin.Manifest.MinServerVersion != "" {
+			_, err := semver.Parse(serverVersion)
+			if err != nil {
+				return nil, errors.Wrapf(err, "failed to parse serverVersion %s", serverVersion)
+			}
```

Comment:
Should this check get moved to after L123 to DRY the code?

Reply:
I tried this but there is some logic of `serverVersion == ""` in both of these blocks, and the `semver.Parse` function throws an error when it is passed an empty string. So I'll keep it how it is unless you see another way to improve the flow.

Reply:
Good point. It's probably fine for now.